src/index.md

```diff
@@ -310,23 +310,26 @@ If the source for the Project is contained within a single repository or reading
 * [JDK Bug System (JBS)](https://bugs.openjdk.java.net/)
 :::
 
-[JBS](https://bugs.openjdk.java.net/) is an issue tracker used by many OpenJDK projects.
+[JBS](https://bugs.openjdk.java.net/) is a public issue tracker used by many OpenJDK projects. It's open for anyone to read and search. In order to get write access you need to be an [Author](https://openjdk.java.net/bylaws#author) in an OpenJDK project.
```

Comment:
For write access, you need to be registered in the OpenJDK Census (for instance by becoming a Project Author).  

[ For membership in an existing Project, there is no requirement that one is an Author before Committer.  For new Projects, the Lead may appoint an individual at any role.  There are no Authors in Groups and Group members also have OpenJDK accounts. ]

Reply:
Fixed.